src/harmony/Dot/Dot.module.css

```diff
@@ -2,11 +2,13 @@
     display: inline-block;
     border-radius: 50%;
     background-color: var(--dot-background);
+    vertical-align: middle;
 }
 
 .DotSmall {
     width: 0.375rem;
     height: 0.375rem;
+    vertical-align: 0.125em;
```

Comment:
magic number

Reply:
I can't explain it, however it correctly works
I saw this magic in [`font-awesome` docs](https://docs.fontawesome.com/web/add-icons/svg-symbols#make-sure-to-add-some-css) and tried to use it